sdk/cosmos/azure-cosmos-test/src/main/java/com/azure/cosmos/test/faultinjection/FaultInjectionServerErrorResult.java

```diff
@@ -14,17 +14,20 @@ public final class FaultInjectionServerErrorResult implements IFaultInjectionRes
     private final Duration delay;
 
     private final Boolean suppressServiceRequests;
+    private final Double applyPercentage;
 
     FaultInjectionServerErrorResult(
         FaultInjectionServerErrorType serverErrorTypes,
         Integer times,
         Duration delay,
-        Boolean suppressServiceRequests) {
+        Boolean suppressServiceRequests,
+        Double applyPercentage) {
```

Comment:
Double in Java is Nullable<double> in .Net - please amke this double (non-nullable) with a default of 1 (100 percent)?